ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86-ovnic.yaml

```diff
@@ -37,9 +37,9 @@ tests:
     pre:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
-    workflow: openshift-qe-cluster-density-v2
+    workflow: openshift-qe-control-plane
```

Comment:
cdv2 alone took 1h45m recently. A rehearse would tell us, but we also likely need to set higher timeouts in the refs to make sure we don't timeout.